.circleci/config.yml

```diff
@@ -156,20 +156,52 @@ jobs:
             git fetch
             git rebase
       - run:
-          name: build images
+          name: build Java 11.0.17 & Node 14.21.1 image
+          command: |
+                MBT_VERSION=$(cat ./VERSION)
+                bash $PWD/scripts/build_image 11.0.17 14.21.1 ${MBT_VERSION}
+      - run:
+          name: build Java 11.0.17 & Node 16.18.1 image
+          command: |
+              MBT_VERSION=$(cat ./VERSION)
+              bash $PWD/scripts/build_image 11.0.17 16.18.1 ${MBT_VERSION}
+      - run:
+          name: build Java 11.0.17 & Node 18.12.1 image
+          command: |
+              MBT_VERSION=$(cat ./VERSION)
+              bash $PWD/scripts/build_image 11.0.17 18.12.1 ${MBT_VERSION}
+      - run:
+          name: build Java 17.0.5 & Node 14.21.1 image
+          command: |
+                MBT_VERSION=$(cat ./VERSION)
+                bash $PWD/scripts/build_image 17.0.5 14.21.1 ${MBT_VERSION}
+      - run:
+          name: build Java 17.0.5 & Node 16.18.1 image
+          command: |
+              MBT_VERSION=$(cat ./VERSION)
+              bash $PWD/scripts/build_image 17.0.5 16.18.1 ${MBT_VERSION}
+      - run:
+          name: build Java 17.0.5 & Node 18.12.1 image
+          command: |
+                MBT_VERSION=$(cat ./VERSION)
+                bash $PWD/scripts/build_image 17.0.5 18.12.1 ${MBT_VERSION}
+      - run:
+          name: build Java 19.0.1 & Node 14.21.1 image
           command: |
               MBT_VERSION=$(cat ./VERSION)
-              sh $PWD/scripts/build_image 11.0.17 14.21.1 ${MBT_VERSION}
-              sh $PWD/scripts/build_image 11.0.17 16.18.1 ${MBT_VERSION}
-              sh $PWD/scripts/build_image 11.0.17 18.12.1 ${MBT_VERSION}
-              sh $PWD/scripts/build_image 17.0.5 14.21.1 ${MBT_VERSION}
-              sh $PWD/scripts/build_image 17.0.5 16.18.1 ${MBT_VERSION}
-              sh $PWD/scripts/build_image 17.0.5 18.12.1 ${MBT_VERSION}
-              sh $PWD/scripts/build_image 19.0.1 14.21.1 ${MBT_VERSION}
-              sh $PWD/scripts/build_image 19.0.1 16.18.1 ${MBT_VERSION}
-              sh $PWD/scripts/build_image 19.0.1 18.12.1 ${MBT_VERSION}
+              bash $PWD/scripts/build_image 19.0.1 14.21.1 ${MBT_VERSION}
       - run:
-          name: publish images
+          name: build Java 19.0.1 & Node 16.18.1 image
+          command: |
+              MBT_VERSION=$(cat ./VERSION)
+              bash $PWD/scripts/build_image 19.0.1 16.18.1 ${MBT_VERSION}
+      - run:
+          name: build Java 19.0.1 & Node 18.12.1 image
+          command: |
+              MBT_VERSION=$(cat ./VERSION)
+              bash $PWD/scripts/build_image 19.0.1 18.12.1 ${MBT_VERSION}
```

Comment:
Please revert to a portable bourne shell, none of the scripts shell used in that repository have bashisms requiring to use bash only.

Reply:
Please push your changes after saying it's done for review. Thanks.